config.json

```diff
@@ -288,296 +307,395 @@
+        "practices": ["strings"],
+        "prerequisites": ["basics", "bools", "conditionals", "strings"],
+        "difficulty": 1
+      },
+      {
+        "slug": "space-age",
+        "name": "Space Age",
+        "uuid": "f8303c4d-bbbb-495b-b61b-0f617f7c9a13",
+        "practices": ["dicts"],
         "prerequisites": [
           "basics",
           "bools",
-          "conditionals",
+          "dicts",
           "lists",
           "list-methods",
           "loops",
-          "strings",
-          "string-methods",
           "numbers"
         ],
-        "difficulty": 4
+        "difficulty": 1
       },
       {
-        "slug": "clock",
-        "name": "Clock",
-        "uuid": "459fda78-851e-4bb0-a416-953528f46bd7",
-        "practices": [
-          "class-composition",
-          "rich-comparisons",
-          "string-formatting"
-        ],
-        "prerequisites": ["basics", "numbers", "strings", "classes"],
-        "difficulty": 4
-      },
-      {
-        "slug": "markdown",
-        "name": "Markdown",
-        "uuid": "88610b9a-6d3e-4924-a092-6d2f907ed4e2",
-        "practices": ["regular-expressions", "functions"],
-        "prerequisites": [],
-        "difficulty": 4
+        "slug": "collatz-conjecture",
+        "name": "Collatz Conjecture",
+        "uuid": "33f689ee-1d9c-4908-a71c-f84bff3510df",
+        "practices": ["numbers"],
+        "prerequisites": ["basics", "numbers"],
+        "difficulty": 1
       },
       {
-        "slug": "tournament",
-        "name": "Tournament",
-        "uuid": "49377a3f-38ba-4d61-b94c-a54cfc9034d0",
-        "practices": [
-          "anonymous-functions",
-          "collections",
-          "dicts",
-          "dict-methods",
-          "generators",
-          "generator-expressions",
-          "other-comprehensions",
-          "string-formatting",
-          "string-methods-splitting",
-          "tuples"
-        ],
+        "slug": "secret-handshake",
+        "name": "Secret Handshake",
+        "uuid": "0d5b2a0e-31ff-4c8c-a155-0406f7dca3ae",
+        "practices": ["list-methods"],
         "prerequisites": [
           "basics",
-          "strings",
-          "string-methods",
-          "dicts",
+          "bools",
+          "conditionals",
           "lists",
           "list-methods",
-          "loops",
-          "tuples"
+          "numbers",
+          "strings",
+          "string-methods"
         ],
-        "difficulty": 5
+        "difficulty": 1
       },
       {
-        "slug": "book-store",
-        "name": "Book Store",
-        "uuid": "4899b2ef-675f-4d14-b68a-1a457de91276",
-        "practices": [
-          "collections",
-          "conditionals",
-          "functools",
-          "generator-expressions",
-          "other-comprehensions",
-          "sets"
-        ],
+        "slug": "wordy",
+        "name": "Wordy",
+        "uuid": "af50bb9a-e400-49ce-966f-016c31720be1",
+        "practices": ["string-methods"],
```

Comment:
Yup....and you can see my delemma.  Because I needed to find exercises that practice `string-methods` (_as opposed to `strings`_) and yet don't require something further down the tree (mostly) to unlock.  Only concerned right now with the active exercises, because we have leeway for more re-org if a concept or exercise is not public (WIP) or TBD.

The other thing we can do is make notes on holes, and then make practice exercises that fulfill that.  It might remove the (high) tension for me around specific exercises like `High Scores` that is really good for `list` or `list-methods` ... but is also really good for basic `class` creation.  And also give us an opportunity to create string and string processing exercises that have nothing to do with cumulative songs...which I (mostly) hate.